context/context.go

```diff
@@ -51,6 +54,18 @@ func newPooledContext(pool contextPool) Context {
 	return &ctx{pool: pool}
 }
 
+func (c *ctx) GoContext() (xnetcontext.Context, bool) {
```

Comment:
Should this be called `GoContext` if it's `golang.org/x/net/context`? I'd expect GoContext to use `context`

Reply:
As Jerome pointed out `golang.org/x/net/context` is just an alias for `context`. Either way, I changed it to just use `context`.